### DISCUSSION TITLE
Can Zeek analyse captured packets in other networks in real time?

### DISCUSSION THREAD

**Discussion by minh-chau** [zeek/zeek#2666] [How-Tos]
I'm a fresh Zeek user. From documentation, Zeek supports to analyses packets from pcap files or sniffing on interfaces. However, I'm using Zeek in a Kubernetes cluster as a dedicated real time packet analysis service. The packets captured from other networks needs to feed into Zeek service in real time. Here the problem comes, does Zeek have a mechanism to take captured packets as input from other source as a stream in real time. My thoughts it would be whether Zeek supports to open a TCP connection in which Zeek can receive the packets and feed them into the event engine in real time, just similar as how Zeek takes the sniffed packets as input.

Regards
Minh

**Comment by ckreibich** [comment]
We don't currently support this but it's been [requested before](https://github.com/zeek/zeek/issues/2348). This'd be a really nice Zeek package — would you be interested in taking this on?
